src/js/auth/containers/LoginApp.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import environment from '../../common/helpers/environment.js';
+
+import { updateLoggedInStatus, updateProfileField } from '../../common/actions';
+
+class LoginApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.setMyToken = this.setMyToken.bind(this);
+  }
+
+  componentDidMount() {
+    this.setMyToken();
+  }
+
+  setMyToken() {
+    const saml = new FormData();
+    saml.append('SAMLResponse', this.props.location.query.SAMLResponse);
+    fetch(`${environment.API_URL}/auth/login/callback`, {
```

Comment:
let's keep this as `/auth/saml/callback` (what's on vets-api master right now) to minimize config churn. Not that this does not have to match the route for the LoginApp react component.

FYI: @omgitsbillryan